Automattic/wp-calypso
# Adds e2e tests for My Home “Get help” support search

Builds on https://github.com/Automattic/wp-calypso/pull/43381 to add e2e tests to cover the support search instance on the "My Home" page (ie: the "Get help" card).

<img width="320" alt="Screenshot 2020-06-24 at 14 18 19" src="https://user-images.githubusercontent.com/444434/85563757-951d6c00-b625-11ea-9815-4dba6c462acb.png">

This also [parallelizes the test runs as per @bsessions85 advice](https://github.com/Automattic/wp-calypso/pull/43625#issuecomment-648977498).


#### Testing instructions

* Configure the Calypso local e2e testing environment [as per the docs](https://github.com/Automattic/wp-calypso/blob/f35d4cc98655259e379fe220be9b9ecee6bac647/test/e2e/README.md).
* Boot Calypso locally - `yarn && yarn start`. Make sure it's running at `http://calypso.localhost:3000/`.
* Run the individual e2e test 
  - `cd` to the `test/e2e` directory.
  - run `./node_modules/.bin/mocha specs/wp-my-home-support-search.js`
* See tests run and verify they all pass.
* Do the same against tablet and mobile using following command syntax - `env BROWSERSIZE=mobile ./node_modules/.bin/mocha specs/wp-support-search`.
* Read through tests and check there are no missing scenarios that might be helpful to cover.
* ⚠️  **Important:** verify that the tests for the FAB support search still continues to pass as well - `./node_modules/.bin/mocha specs/wp-inline-help-support-search.js`.

Fixes https://github.com/Automattic/wp-calypso/issues/43135
